sdk/core/azure-core/CMakeLists.txt

```diff
@@ -53,16 +63,23 @@ add_library (
   src/version.cpp
   )
 
-target_include_directories (${TARGET_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc> $<INSTALL_INTERFACE:include/az_core>)
+target_include_directories(
+  ${TARGET_NAME}
+    PUBLIC
+      $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
+      $<INSTALL_INTERFACE:include>
+  )
 
 # make sure that users can consume the project as a library.
-add_library (Azure::Core ALIAS ${TARGET_NAME})
+add_library(Azure::${TARGET_NAME} ALIAS ${TARGET_NAME})
 
 # coverage. Has no effect if BUILD_CODE_COVERAGE is OFF
 create_code_coverage(core ${TARGET_NAME} "${TARGET_NAME}-test")
 
 target_include_directories(${TARGET_NAME} PUBLIC ${CURL_INCLUDE_DIRS})
-target_link_libraries(${TARGET_NAME} PRIVATE CURL::libcurl Threads::Threads)
+target_link_libraries(${TARGET_NAME} PUBLIC CURL::libcurl Threads::Threads)
```

Comment:
why are you changing this?
libcurl and threads libraries are consumed only by Azure CORE.  By doing PUBLIC, we are saying that the Azure SDK also provides this 2 with it

Reply:
But it does! We require customer to link with it.